### PR TITLE
[api][SIMS - #File/sims-#646] Receive BC E-Cert Feedback File

### PR DESCRIPTION
**Pull request by ann-aot**
added code to proceed silently if there is a unique constraint query error

### CODE REVIEW

**Review by dheepak-aot — APPROVED** [review]
LGTM. 👍 Please connect the issue to PR.

File: sources/packages/api/src/services/disbursement-schedule-errors/disbursement-schedule-errors.service.ts [hunk 1]
```diff
@@ -36,6 +36,14 @@ export class DisbursementScheduleErrorsService extends RecordDataModelService<Di
...
       return newDisbursementsFeedbackErrors;
     });
-    return this.repo.save(errorCodesObject);
+    return this.repo
+      .createQueryBuilder()
+      .insert()
+      .into(DisbursementFeedbackErrors)
+      .values(errorCodesObject)
+      .onConflict(
+        `ON CONSTRAINT disbursement_schedule_id_error_code_unique DO NOTHING`,
```
Comment (andrewsignori-aot) [inline sources/packages/api/src/services/disbursement-schedule-errors/disbursement-schedule-errors.service.ts:45]: Don't need the interpolation back ticks if nothing is being interpolated 😉

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

File: sources/packages/api/src/services/disbursement-schedule-errors/disbursement-schedule-errors.service.ts (hunk 1 above)
Comment (ann-aot) [reply sources/packages/api/src/services/disbursement-schedule-errors/disbursement-schedule-errors.service.ts]: updated

**Pull request merged**